types/atom/index.d.ts

```diff
@@ -3610,7 +3619,10 @@ export class GitRepository {
 /** Grammar that tokenizes lines of text. */
 export interface Grammar {
     /** The name of the Grammar. */
-    name: string;
+    readonly name: string;
```

Comment:
I think this change makes sense, as I can't really think of any reason why the internal properties of a Grammar would ever need to be modified after instantiation... it seems to me that this would likely only break things, so making it `readonly` seems appropriate. This can always be modified if it turns out there is a legitimate reason to make these properties writable.